meshnet/server.go

```diff
@@ -3069,6 +3069,22 @@ func (s *Server) getPeerWithIdentifier(id string, peers mesh.MachinePeers) *mesh
 	return &peers[index]
 }
 
+func (s *Server) listPeers() (mesh.MachinePeers, error) {
+	var cfg config.Config
+	if err := s.cm.Load(&cfg); err != nil {
+		log.Println(internal.ErrorPrefix, "failed to read configuration when listing peers:", err)
+		return nil, fmt.Errorf("reading configuration when listing peers: %w", err)
+	}
+
+	token := cfg.TokensData[cfg.AutoConnectData.ID].Token
+	peers, err := s.reg.List(token, cfg.MeshDevice.ID)
+	if err != nil {
+		log.Println(internal.ErrorPrefix, "failed to list peers:", err)
```

Comment:
It might be better not to print logs here.
Since the function returns an error, the caller should decide what to do with the error. Otherwise there will be duplicate messages into logs.
Also above.

Reply:
Done :+1: